src/views/ReviewConfirm.vue

```diff
@@ -33,23 +33,28 @@
 
     <section class="mt-10">
       <header>
-        <h2>Completing Party Statement</h2>
+        <h2>Certify</h2>
+        <p class="mt-1">Enter the legal name of the person authorized to complete and submit this application.</p>
       </header>
-      <certify
-        :currentDate="currentDate"
-        :certifiedBy="getCertifyState.certifiedBy"
-        :isCertified="getCertifyState.valid"
-        :statements="getCompletingPartyStatement.certifyStatements"
-        :message="getCompletingPartyStatement.certifyClause"
-        :enableMailTo="true"
-        :businessEmail="getBusinessEmail"
-        :completingPartyEmail="getCompletingPartyEmail"
-        :isStaff="isRoleStaff"
-        :firstColumn="3"
-        :secondColumn="9"
-        @update:certifiedBy="onCertifiedBy($event)"
-        @update:isCertified="onIsCertified($event)"
-      />
+      <div :class="{ 'invalid-section': isCertifyInvalid }">
+        <certify
+          :currentDate="currentDate"
+          :certifiedBy="getCertifyState.certifiedBy"
+          :isCertified="getCertifyState.valid"
+          :statements="getCompletingPartyStatement.certifyStatements"
+          :message="getCompletingPartyStatement.certifyClause"
+          :enableMailTo="true"
+          :businessEmail="getBusinessEmail"
+          :completingPartyEmail="getCompletingPartyEmail"
+          :isStaff="isRoleStaff"
+          :firstColumn="3"
+          :secondColumn="9"
+          :invalidSection="isCertifyInvalid"
+          :disableEdit="!isRoleStaff"
```

Comment:
Does this new prop mean the "isStaff" functionality of Certify is no longer needed? (UX question, I know.)

Reply:
I do not know know, potentially though. 
I thought it might be more useful to disable the input and let the app decide vs adding more staff rules into the component itself. 

isStaff seems to just change the statement text very slightly 🤷